samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -57,24 +60,27 @@ Models
 
 The following table shows the Bluetooth mesh sensor composition data for this sample:
 
-   +---------------+
-   |  Element 1    |
-   +===============+
-   | Config Server |
-   +---------------+
-   | Health Server |
-   +---------------+
-   | Sensor Server |
-   +---------------+
+   +---------------------+
+   | Element 1           |
+   +=====================+
+   | Config Server       |
+   +---------------------+
+   | Health Server       |
+   +---------------------+
+   | Sensor Server       |
+   +---------------------+
+   | Sensor Setup Server |
+   +---------------------+
 
 The models are used for the following purposes:
 
 * Config Server allows configurator devices to configure the node remotely.
 * Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
   These callbacks trigger blinking of the LEDs.
 * Sensor Server provides sensor data to one or more :ref:`mesh sensor observers <bt_mesh_sensor_cli_readme>`.
+* Sensor Setup Server is used for configuration of the Sensor Server.
 
-The model handling is implemented in :file:`src/model_handler.c`, which uses the ``TEMP_NRF5`` temperature sensor, and the :ref:`dk_buttons_and_leds_readme` library to detect button presses.
+The model handling is implemented in :file:`src/model_handler.c`, which uses the ``TEMP_NRF5`` or ``BME680`` temperature sensor depending on the platform, and the :ref:`dk_buttons_and_leds_readme` library to detect button presses.
```

Comment:
This was a long sentence. Could we write something like this instead:
"The model handling is implemented in :file:`src/model_handler.c`. It uses the ``TEMP_NRF5`` or ``BME680`` temperature sensor depending on the platform. The :ref:`dk_buttons_and_leds_readme` library is used to detect button presses."